src/components/sbb-expansion-panel-header/sbb-expansion-panel-header.tsx

```diff
@@ -27,7 +27,7 @@ import { resolveButtonRenderVariables } from '../../global/interfaces';
   styleUrl: 'sbb-expansion-panel-header.scss',
   tag: 'sbb-expansion-panel-header',
 })
-export class SbbExpansionPanelHeader implements ComponentInterface {
+export class SbbExpansionPanelHeader implements ButtonProperties, ComponentInterface {
```

Comment:
`hostAttributes['aria-disabled'] = this.disabled?.toString();` : why is this needed? (the true case would already be rendered by resolveButtonVariables) @DavideMininni-Fincons